frontend/src/metabase/dashboard/actions/actions.ts

```diff
@@ -21,16 +19,12 @@ import { getDashboardType } from "../utils";
 import { setDashCardAttributes } from "./core";
 import { closeSidebar, setSidebar } from "./ui";
 
-interface DashboardAttributes {
-  card_id?: CardId | null;
-  action?: WritebackAction | null;
-  parameter_mappings?: ActionParametersMapping[] | null;
-  visualization_settings?: ActionDashboardCard["visualization_settings"];
-}
-
 export function updateButtonActionMapping(
   dashCardId: number,
-  attributes: DashboardAttributes,
+  attributes: Pick<
```

Comment:
I'd not inline such type, previous version was more readable

Reply:
Updated [here](https://github.com/metabase/metabase/pull/39472/commits/1e7be5ddb9ee20c92e040ff9b73e094855ca15dc) in the next PR in a chain (#39472)